Week3/vehicleList.js

```diff
@@ -15,39 +15,56 @@ motorbikes, caravans and bikes.". (Hint: use a for loop.)
 What if you add one more vehicle to the list, can you have that added to the advertisement without changing the code for question 10?
  */
 
-console.log(`------- 7 / 8 / 9 / 10 / 11 -------`)
+console.log(`------- 7 / 8 / 9 / 10 / 11 -------`);
 console.log(`\n`);
 
 const vehicleList = [`car`, `motorbike`, `caravan`, `bike`, `scooter`];
 console.log(`The third element of the vehicle list is: ${vehicleList[2]}`);
 
 function vehicle(color, vehicleIndex, age) {
   let status;
+  let type;
 
-  if (age < 5) {
+  if (age >= 0 && age < 5) {
     status = `new`;
-  } else {
+  } else if (age >= 5) {
     status = `used`;
+  } else {
+    console.log(`Error: The age can not be smaller than 0.`);
+    return;
+  }
+
+  if (vehicleIndex < 0 || vehicleIndex > vehicleList.length) {
+    console.log(
+      `Error: vehicleIndex (${vehicleIndex}) can not be smaller than 0 or bigger than the length of vehicleList (${
+        vehicleList.length
+      })!`,
+    );
+    return;
+  } else {
+    type = vehicleList[vehicleIndex];
```

Comment:
Ah nice!:) also nice error message

Reply:
It was very useful feedback thanks again.